src/platform/windows/ipc/process_handler.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
```

Comment:
All header and source files should have a top level doxygen block.

include statements should be under a comment like:

- `// standard includes`
- `// local includes`
- `// platform includes`

You can look at existing files to see our existing patterns.